core-dev/packages/coq.8.9.dev/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "1.2"
+maintainer: "coqdev@inria.fr"
+authors: "The Coq development team, INRIA, CNRS, University Paris Sud, University Paris 7, Ecole Polytechnique."
```

Comment:
This author list is outdated compared to the one that is present, e.g. in `coq.opam` in the Coq repo...

Reply:
Fixed.